src/views/virtualmachines/tree/hooks/useTreeviewItemRightClick.tsx

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useLayoutEffect, useState } from 'react';
+
+import { TreeViewDataItem } from '@patternfly/react-core';
+
+type UseTreeviewItemRightClickType = (
```

Comment:
nitpick: `UseTreeviewItemRightClickType` -> `UseTreeViewItemRightClick`

It looks the majority of hook types in the code base don't have 'Type' at the end, so I would drop it here to remain consistent.